src/main/java/io/github/communitymod/core/util/enums/ModToolMaterials.java

```diff
@@ -0,0 +1,55 @@
+package io.github.communitymod.core.util.enums;
+
+import java.util.function.Supplier;
+
+import io.github.communitymod.core.init.ItemInit;
+import net.minecraft.util.LazyLoadedValue;
+import net.minecraft.world.item.Tier;
+import net.minecraft.world.item.crafting.Ingredient;
+
+@SuppressWarnings("deprecation")
```

Comment:
What here is deprecated?

Reply:
LazyLoadedValue or something. I wouldn't use that anymore, I'd just replace with th

Reply:
what can i replace LazyLoadedValue with?

Reply:
you could just replace it with a supplier tbh

Reply:
I have `this.repairIngredient = Suppliers.memoize(repairIngredient::get);`

Reply:
what do you have when you initialize the variables for the repair ingredient?